test/controllers/about_controller_test.exs

```diff
@@ -1,8 +1,31 @@
 defmodule ResearchResource.AboutControllerTest do
   use ResearchResource.ConnCase
 
+  setup %{conn: conn} = config do
+    if username = config[:login_as] do
+      user = insert_user(email: username)
+      conn = assign(build_conn(), :current_user, user)
+      {:ok, conn: conn, user: user}
+    else
+      :ok
+    end
+  end
+
```

Comment:
I see what you mean with the `ConnCase` issue, I've implemented a helper function which takes the needed functions as arguments.

Let me know what you think